src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -152,22 +152,36 @@ into
 
 === Database configuration
 
-Scheduler, Resource Mananger, and three microservices require to have direct access to the database.
+Scheduler, Resource Mananger, and four microservices require to have direct access to the database.
```

Comment:
Why not saying : "Scheduler, Resource Mananger, and some microservices ..." In order to not update this number after each new created  micro service.